packages/api/config/workflowLambdas.yml

```diff
@@ -1,26 +1,9 @@
-sf2snsStart:
-  handler: index.start
-  timeout: 100
+publishReports:
```

Comment:
Ah, my bad. I'll remove from `api/config/lamdas.yml`. It needs to be defined as a "workflow" Lambda though so that it can be used in the Parse PDR workflow as the "PdrStatusReport" step

Reply:
e075ee0